pkgs/by-name/fw/fwupd/package.nix

```diff
@@ -84,17 +84,6 @@
 let
   isx86 = stdenv.hostPlatform.isx86;
 
-  # Dell isn't supported on Aarch64
-  haveDell = isx86;
-
-  # only redfish for x86_64
-  haveRedfish = stdenv.hostPlatform.isx86_64;
-
-  # only use msr if x86 (requires cpuid)
-  haveMSR = isx86;
-
-  # # Currently broken on Aarch64
-  # haveFlashrom = isx86;
```

Comment:
What happened to these?
I'm on mobile, so I couldn't check the upstream diff yet.

Reply:
Sorry, just saw the discussion above now. These were removed upstream?

Reply:
- https://github.com/fwupd/fwupd/commit/587b5b177d6ff50aa4d458b726f590f9bafbba6c
 - https://github.com/fwupd/fwupd/commit/5bff80d05c610963a34744411896bf13ef06ec48